### PR TITLE
add yahoofinance growth tech & most active to disc

### PR DESCRIPTION
**Pull request by devwolff**

### CODE REVIEW

File: gamestonk_terminal/stocks/discovery/disc_controller.py [hunk 1]
```diff
@@ -81,6 +83,8 @@ def print_help():
     gainers        show latest top gainers
     losers         show latest top losers
     under          undervalued growth stocks
+    gtech          highest growth technology stocks
```
Comment (DidierRLopes) [inline gamestonk_terminal/stocks/discovery/disc_controller.py:86]: Can we have:
```
gtech       technology stocks with revenue and earnings growth in excess of 25%.
active       most active stocks by intraday trade volume.
```

Would be good to have the description be a bit more accurate as explained in here: https://finance.yahoo.com/screener.

File: gamestonk_terminal/stocks/discovery/disc_controller.py [hunk 2]
```diff
@@ -352,6 +356,90 @@ def call_under(self, other_args: List[str]):
         except Exception as e:
             print(e, "\n")
 
+    def call_gtech(self, other_args: List[str]):
+        """Process gtech command"""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="gtech",
+            description="Print up to 25 top ticker losers. [Source: Yahoo Finance]",
```
Comment (DidierRLopes) [inline gamestonk_terminal/stocks/discovery/disc_controller.py:365]: You forgot to update this description. I would pick the description from here https://finance.yahoo.com/screener

File: gamestonk_terminal/stocks/discovery/disc_controller.py [hunk 3]
```diff
@@ -352,6 +356,90 @@ def call_under(self, other_args: List[str]):
...
+        try:
+            if other_args:
+                if "-" not in other_args[0]:
+                    other_args.insert(0, "-n")
+
+            ns_parser = parse_known_args_and_warn(parser, other_args)
+            if not ns_parser:
+                return
+
+            yahoofinance_view.display_gtech(
```
Comment (DidierRLopes) [inline gamestonk_terminal/stocks/discovery/disc_controller.py:393]: Perfect.

Unrelated, I wrote as a convention on the contributing document that we could have:
`plot_` for a plot
`display_` for a table
`print_` for a string

What do you think?

File: gamestonk_terminal/stocks/discovery/disc_controller.py [hunk 4]
```diff
@@ -352,6 +356,90 @@ def call_under(self, other_args: List[str]):
...
+        except Exception as e:
+            print(e, "\n")
+
+    def call_active(self, other_args: List[str]):
+        """Process active command"""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="active",
+            description="Print up to 25 top ticker losers. [Source: Yahoo Finance]",
```
Comment (DidierRLopes) [inline gamestonk_terminal/stocks/discovery/disc_controller.py:407]: forgot to update this

File: gamestonk_terminal/stocks/discovery/disc_controller.py [hunk 5]
```diff
@@ -352,6 +356,90 @@ def call_under(self, other_args: List[str]):
...
+        )
+        parser.add_argument(
+            "-n",
+            "--num",
+            action="store",
+            dest="num",
+            type=check_int_range(1, 25),
+            default=5,
+            help="Number of the undervalued stocks to retrieve.",
```
Comment (DidierRLopes) [inline gamestonk_terminal/stocks/discovery/disc_controller.py:374]: Change to "Number of stocks to display."

File: gamestonk_terminal/stocks/discovery/disc_controller.py [hunk 6]
```diff
@@ -352,6 +356,90 @@ def call_under(self, other_args: List[str]):
...
```
Comment (DidierRLopes) [inline gamestonk_terminal/stocks/discovery/disc_controller.py:416]: Change to "Number of stocks to display.",

File: gamestonk_terminal/stocks/discovery/yahoofinance_model.py [hunk 7]
```diff
@@ -44,3 +44,32 @@ def get_undervalued() -> pd.DataFrame:
     data = pd.read_html(requests.get(url).text)[0]
     data = data.iloc[:, :-1]
     return data
+
+
+def get_gtech() -> pd.DataFrame:
+    """Get Yahoo Finance growth technology stocks
```
Comment (DidierRLopes) [inline gamestonk_terminal/stocks/discovery/yahoofinance_model.py:50]: Can we have:
"""Get Technology stocks with revenue and earnings growth in excess of 25%. [Source: Yahoo Finance]

File: gamestonk_terminal/stocks/discovery/yahoofinance_model.py [hunk 8]
```diff
@@ -44,3 +44,32 @@ def get_undervalued() -> pd.DataFrame:
...
+    """
+    url = "https://finance.yahoo.com/screener/predefined/growth_technology_stocks"
+
+    data = pd.read_html(requests.get(url).text)[0]
+    data = data.iloc[:, :-1]
+    return data
+
+
+def get_active() -> pd.DataFrame:
+    """Get Yahoo Finance most active stocks
```
Comment (DidierRLopes) [inline gamestonk_terminal/stocks/discovery/yahoofinance_model.py:65]: Similar as above. I.e. description from here https://finance.yahoo.com/screener with the [Source: Yahoo Finance] at the end

File: gamestonk_terminal/stocks/discovery/disc_controller.py (hunk 3 above)
Comment (devwolff) [reply gamestonk_terminal/stocks/discovery/disc_controller.py:393]: I think it's a great idea!

**Comment by devwolff** [comment]
fix them asap

**Comment by DidierRLopes** [comment]
> ill fix them asap

I'm gonna go to bed now because I need to wake up in 4 hours 😭  But when I wake up I'll merge this!!

**Comment by devwolff** [comment]
🥲 try get some more

**Review by DidierRLopes — APPROVED** [review]

**Pull request merged**